app/presenters/finder_presenter.rb

```diff
@@ -86,16 +86,15 @@ def show_summaries?
     content_item.details.show_summaries
   end
 
-  def organisations
-    content_item.links.organisations
-  end
-
-  def part_of
-    @part_of = content_item.links.part_of || []
+  def any_page_metadata?
+    page_metadata.any? { |type, links| links.any? }
```

Comment:
Would it be better to bake this into the return value of `page_metadata` instead? Eg

```ruby
def page_metadata
  metadata = {
    part_of: part_of,
    from: from,
  }

  metadata.reject { |_, links| links.empty? }
end
```

That way, you could just use `page_metadata.any?` for this method, and other consumers of `page_metadata` don't have to worry about some of the values being empty arrays.

Relatedly, if you have a variable you aren't using (`type` in this case) it's good practice to use `_` as the variable, or prefix it like `_type` if you want to give context on what it is but indicate it's unused.